packages/www/pages/register.tsx

```diff
@@ -36,21 +39,32 @@ const RegisterPage = () => {
     organization,
     phone,
   }) => {
-    const selectedPlan = router.query?.selectedPlan;
-    setLoading(true);
-    setErrors([]);
-    const res = await register({
-      email,
-      password,
-      selectedPlan: selectedPlan ? +selectedPlan : 0,
-      ...(firstName && { firstName }),
-      ...(lastName && { lastName }),
-      ...(organization && { organization }),
-      ...(phone && { phone }),
-    });
-    // Don't need to worry about the success case, we'll redirect
-    if (res.errors) {
-      setErrors(res.errors);
+    if (!executeRecaptcha) {
+      console.log("Execute recaptcha not yet available");
+      return;
+    }
+    try {
+      const recaptchaToken = await executeRecaptcha("register");
+      const selectedPlan = router.query?.selectedPlan;
+      setLoading(true);
+      setErrors([]);
+      const res = await register({
+        email,
+        password,
+        selectedPlan: selectedPlan ? +selectedPlan : 0,
+        ...(firstName && { firstName }),
+        ...(lastName && { lastName }),
+        ...(organization && { organization }),
+        ...(phone && { phone }),
+        recaptchaToken,
+      });
+      // Don't need to worry about the success case, we'll redirect
+      if (res.errors) {
+        setErrors(res.errors);
+        setLoading(false);
+      }
+    } catch (err) {
```

Comment:
Seems like we should either remove this block or change it to `setErrors` so we don't just fail silently